proposals/README.md

```diff
@@ -0,0 +1,28 @@
+# BuildingSync Feature Proposals #
+
+## Overview ##
+This document lays out the procedure for the proposal and completion of changes to BuildingSync. The procedure is described in further detail below. The proposal process is initiated with the creation of a document similar to this one. The document is expected to have at least three sections: **Overview**, **Justification**, and **Implementation**. The overview section provides a high-level description of the proposal, the justification section provides evidence that the changes are needed or required, and the implementation section describes how the changes will be implemented. 
+
+## Justification ##
+BuildingSync is intended to be a standardized language for commercial building energy audit data that software developers can use to exchange data between audit tools. BuildingSync will only continue to provide a standard data specification for audit-related information if there is an open, transparent process by which stakeholders can propose changes, in particular additions, to the schema.
+
+## Implementation ##
+The process is semi-formal and is not intended to be rigidly applied. The procedure is roughly comprised of the following steps:
+
+  1. The proposer writes a Markdown document, modeled on this document, that succinctly describes the changes that are proposed.
+  2. The proposer creates a GitHub pull request that adds the document to the proposal directory in a subdirectory named for the year in which the submission is being made (e.g. proposals/2018/Add_New_Feature_XYZ.md).
+  3. The proposer, the BuildingSync team, and other stakeholders and interested parties discuss the proposal and make modifications using the GitHub pull request discussion facility. At the conclusion of this process, the proposal is accepted or rejected.
+  4. If the proposal is accepted, it is merged and the appropriate GitHub tracking procedure (issue, project, etc.) is initiated to track progress toward completion.
+  5. The change is made by the appropriate developer (not necessarily the proposer or a BuildingSync team member) following the contribution policy.

```

Comment:
For #5 and #6, my only question would be how we determine who the appropriate developer is.  I do agree that it's important to specify that this person isn't necessarily the proposer or a team member, though.

Reply:
@fieldkm Ideally, a lot of the time it would be the proposer. But I can think of cases where it would need to be someone else (lack of availability, etc.). I guess in cases where it would not be the proposer doing the changes, the "who" should be decided in an earlier step. I could include "identification of the appropriate developer" as part of item 4, before merging. How about:

  4. If the proposal is accepted, a developer is identified (not necessarily the proposer or a BuildingSync team member) and the pull request is merged. The appropriate GitHub tracking procedure (issue, project, etc.) is initiated to track progress toward completion.

I know that leaves the "how" a little murky, but I think that's OK in this case if selection is required before the proposal is finally accepted.

Reply:
I agree - this is a good compromise.  It probably wouldn't work well to anticipate every possible appropriate developer anyway.  Your solution here looks good to me.